### DISCUSSION TITLE
Reply count for issues

### DISCUSSION THREAD

**Discussion by sydbarrett74** [community/community#122914] [Mobile]
### Select Topic Area

Product Feedback

### Body

It would be nice if the reply count for issues were visible in the mobile UI.

![Screenshot 2024-05-08 022156](https://github.com/community/community/assets/6467403/b8fa0835-cdbd-4a3e-a18d-e4f8c7551376)

**Comment by kangarruu** [comment]
👋🏻 @sydbarrett74 would you mind elaborating a bit more on where you'd expect to see the comment count? Issue comments are currently displayed as part of the metadata in issue lists on the Mobile apps. But if there are no comments that pill would be omitted. Is that what you were referring to? 

![WhatsApp Image 2024-05-08 at 10 20 23 AM](https://github.com/community/community/assets/36139667/1ee2db82-c322-49b6-8832-ff71ccc6d727)

**Comment by sydbarrett74** [comment]
@kangarruu I saw a repo where this issue comment bubble wasn't visible on mobile. I will gather a couple of screenshots this weekend and provide them for you. :)
